src/adr/0042-james-cli-based-on-webadmin.md

```diff
@@ -34,16 +34,14 @@ We decided to write a new CLI client, running on top of the JVM, communicating w
 
   * server/protocols/webadmin-cli
 
-* Distribute this CLI as a GraalVM native image.
-
 * Write a man page.
 
   * Picocli generates beautiful documentation for our CLI (HTML, PDF and Unix man pages).
 
 * We decided to adopt a more modern, modular CLI syntax:
 
 ```   
-$ james-cli [OPTION] ENTITY ACTION {ARGUMENT}
+$ java -jar james-cli.jar [OPTION] ENTITY-ACTION {ARGUMENT}
```

Comment:
No I disagree with this change.

By doing this you fall back to the previous syntax.

I can no longer have details for the actins permitted on a iven entity like `james-cli domain --help`... And this is a major drawback.

Reply:
>No I disagree with this change.
>By doing this you fall back to the previous syntax.
I agreed with you, hope with bash script we would solve this problem "At first, i designed domain list as a subcommand. But with that, when run jar file the syntax must be james-cli "domain list""

>I can no longer have details for the actins permitted on a iven entity like james-cli domain --help... And this is a major drawback.

Picocli don't support --help option for subcommand( here is domain entity)

Reply:
Can't we have a DomainCommand subcommand having itself a DomainListCommand subcommand?

> Picocli don't support --help option for subcommand( here is domain entity)

If that's `james-cli domain help` or `james-cli domain list help` I'm fine with it too ^^

Reply:
> Can't we have a DomainCommand subcommand having itself a DomainListCommand subcommand?

I am going to change as this too.
> If that's `james-cli domain help` or `james-cli domain list help` I'm fine with it too ^^

How about using built-in HelpCommand (https://picocli.info/#_built_in_help_subcommand) for this specific help, and not using --help options anymore?

Reply:
> How about using built-in HelpCommand (https://picocli.info/#_built_in_help_subcommand) for this specific help, and not using --help options anymore?

:+1: